### ISSUE TITLE
404 on search

### ISSUE THREAD

**Issue by samuba** [Jamedjo/Raticon#1]
This is exactly the tool I was looking for. Thanks for making this! :-)
But right now the search is broken. it always responds with 404

**Comment by MMedhat94** [comment]
Same problem here

**Comment by DineshSolanki** [comment]
yeah, *that was the perfect tool* for what I wanted but it's _broken_ at the time,
so I thought of creating my own tool similar to Raticon's Functionality,
it's currently WIP, and soon will be ready

**Comment by DineshSolanki** [comment]
so, guys, I have almost made it, and I need some help, I am not currently making it public because I'm using devexpress and for you people to edit it, you'll need to get devexpress too, so tell me if anyone is willing to help, I'll contact with the source

![capture](https://user-images.githubusercontent.com/15937452/48302250-5e621300-e520-11e8-932a-00a8631abcc0.PNG)
![auto mode](https://user-images.githubusercontent.com/15937452/48302255-7cc80e80-e520-11e8-88cd-431fa0be6ae5.PNG)
![search](https://user-images.githubusercontent.com/15937452/48302256-7d60a500-e520-11e8-9483-7d18d1c96f21.png)
and there's More...

**Comment by realgethabawy** [comment]
Error: The remote server returned an error: (404) Not Found.

**Comment by Arrow2001** [comment]
@dinesh-solanki I'm willing to help if you still need it but what programming language are you using for it?

**Comment by DineshSolanki** [comment]
Yeah,I could use your help, I'm using vb.net/c#

On Thursday, May 16, 2019, Iain <notifications@github.com> wrote:
> @dinesh-solanki I'm willing to help if you still need it but what
programming language are you using for it?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.<
https://ci5.googleusercontent.com/proxy/f8goLIaUtqZCN2HQz333ncvkvqqrM7ZCF-t00UsCnIYJtBcKKjGMzSfqQvRu8dRAfdq0PyryBYbzQyc1Gn1WvIELBpnht2Ump4NquieLlNqbskrvD1jNogCwqwEOcDzB_0szd1HDBS73GOQ8m17FbCYNvE_UHsDMpfziuci0bICQPReInYx5K9TXzj9OIyvjIYAR5_VeKd2sWqzWu9ij0iGfQ0Zt1m-nJNIxjMDXqQ=s0-d-e1-ft#https://github.com/notifications/beacon/ADZS7LA5TCJAPRX7NEVVH6DPVV2FLA5CNFSM4EKR6K62YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODVSDSVY.gif
>

-- 
<https://about.me/dineshsolanki?promo=email_sig&utm_source=product&utm_medium=email_sig&utm_campaign=gmail_api&utm_content=thumb>
Dinesh Solanki
about.me/dineshsolanki
<https://about.me/dineshsolanki?promo=email_sig&utm_source=product&utm_medium=email_sig&utm_campaign=gmail_api&utm_content=thumb>

**Comment by Arrow2001** [comment]
@dinesh-solanki I don't have much knowledge of C# (other than discord bot) but I don't mind learning more to help.

**Comment by DineshSolanki** [comment]
@Arrowerse2001 CURRENTLY I'm using Google's search API to obtain ready-made folder icons and TMDB API to fetch posters which then I convert to folder icon using the code of @Jamedjo with some modification.
the problem is that the API is linked to my account thus there's a risk of exposing my credentials if I share the working code, second it has a limit, 100 queries per day and fetching more than one page also counts as a query, so it's not sufficient. if you have anything to suggest, I'll be glad.

**Comment by DineshSolanki** [comment]
@Arrowerse2001 and I would share the code but it uses some paid controls and won't work for everyone, so I was thinking to migrate it to native .NET controls and then share the source.

I will suggest you learn .net development, WinForms or WPF for this project, once you do that, you can help me port the source to native controls.

**Comment by DineshSolanki** [comment]
![image](https://user-images.githubusercontent.com/15937452/57969447-add44080-7994-11e9-8001-1fae9cb2e35e.png)

This was done using FoliCon

**Comment by Arrow2001** [comment]
@dinesh-solanki Sure, I can start learning them after Wednesday and by what you said, you are using Google API to obtain ready made folder icons, well I started experimenting and found this https://www.deviantart.com/developers/ as I used that to produce this below. They have hundreds of premade folder icons. 

NOTE: Schooled folder icon is not from deviantart.com hence why it looks strange. 
![image](https://user-images.githubusercontent.com/45532247/57971781-6fd21f00-798a-11e9-9922-34d24376f3b4.png)

**Comment by DineshSolanki** [comment]
By using google api, i meant i'm using google custom search api which will
search on my specified websites, which also includes deviantArt. So it does
fetches from it.
I thought of using deviantArt api, but last when i checked, they didn't had
a sophisticated api that could be used to search and fetch.
But i will have a look into that again.
Thank you, keep researching and learning

**Comment by DineshSolanki** [comment]
Hey everyone,
I nearly ported the project to Native Controls with all basic features working, Remaining features will be introduced soon.
Please have a look at [FoliCon](https://github.com/dinesh-solanki/Folicon)

Feedback, suggestions, and bugs report are welcome

**Comment by DineshSolanki** [comment]
hey everyone reading it, I made a similar software, it's fully functional and running, find it here [FoliCon](https://github.com/dinesh-solanki/Folicon)
